apps/scheduling/models.py

```diff
@@ -1,438 +0,0 @@
-from __future__ import unicode_literals
```

Comment:
Need to compare this with the new version, might have some unwanted model changes.